service-catalog/apis/pulp.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: pulp
+  description: The pulp api of our operate first instance
+spec:
+  type: openapi
+  lifecycle: production
+  owner: group:operate-first
```

Comment:
@fao89 while this is not an issue, maybe we can consider creating a new user group for the Pulp maintainers? It may be different point of contact than the generic Operate First SRE group. :slightly_smiling_face: (I'm fine with leaving this as is, just noting that for your considerations)

Reply:
how can we create a new group?